Resources/AzureDevOps/azure-pipelines.yml

```diff
@@ -0,0 +1,39 @@
+variables:
+    ArmTemplateRoot: '$(Build.SourcesDirectory)\Resources\ArmTemplates'
+    SolutionBaseName: 'DFC.CourseDirectory.SharedResources'
+
+resources:
+  repositories:
+  - repository: self
+  - repository: dfc-devops
+    type: github
+    name: SkillsFundingAgency/dfc-devops
+    ref: refs/tags/v1.6.5
+    endpoint: 'GitHub (ESFA)'
+
+jobs:
+
+#Build and test resources
+- template: AzureDevOpsTemplates/Build/dfc-arm-build.yml@dfc-devops
+  parameters:
+    ArmTemplateRoot: $(ArmTemplateRoot)
+
+#Publish templates
+- job: BuildDotNetCore
+  displayName: Build-DotNetCore
+  dependsOn: TestArmTemplate
+  pool:
+    name: DAS - Continuous Deployment
+
+  steps:
+  - task: CopyFiles@2
```

Comment:
Doesn't the dfc-arm-build template already do this step?

Reply:
Yes,  https://github.com/SkillsFundingAgency/dfc-devops/blob/master/AzureDevOpsTemplates/Build/dfc-arm-build.yml#L34

So, the Copy isn't required, but we do need to publish the artifact.

Reply:
They're different jobs so it doesn't carry over